cmd/subnetcmd/addValidator.go

```diff
@@ -83,15 +80,36 @@ func addValidator(cmd *cobra.Command, args []string) error {
 
 	if network == models.Undefined {
 		networkStr, err := app.Prompt.CaptureList(
-			"Choose a network to deploy on. This command only supports Fuji currently.",
-			[]string{models.Fuji.String(), models.Mainnet.String() + " (coming soon)"},
+			"Choose a network to add validator on.",
+			[]string{models.Fuji.String(), models.Mainnet.String()},
 		)
 		if err != nil {
 			return err
 		}
 		network = models.NetworkFromString(networkStr)
 	}
 
+	switch network {
+	case models.Fuji:
+		if !useLedger && keyName == "" {
+			var err error
+			useLedger, err = app.Prompt.CaptureYesNo("Use ledger as private key to issue the transaction?")
```

Comment:
Instead of a yes/no, could we make this a list to choose key vs ledger and make key the first option?